src/main/java/uk/gov/beis/els/categories/washingmachines/model/WasherDryerForm.java

```diff
@@ -0,0 +1,196 @@
+package uk.gov.beis.els.categories.washingmachines.model;
+
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotBlank;
+import javax.validation.groups.Default;
+import org.hibernate.validator.constraints.Range;
+import uk.gov.beis.els.categories.common.StandardTemplateForm20Char;
+import uk.gov.beis.els.categories.internetlabelling.model.InternetLabellingGroup;
+import uk.gov.beis.els.model.meta.DualModeField;
+import uk.gov.beis.els.model.meta.FieldPrompt;
+import uk.gov.beis.els.model.meta.StaticProductText;
+
+@StaticProductText("You must attach the label to the front or top of the product so that it’s easy to see. It must be at least 96mm x 192mm when printed.")
```

Comment:
This needs the same text about built-in appliances we have for dishwashers - "If it's a built-in washer-dryer it doesn't have to be attached to the product, but it must still be easy to see."